src/care_radiology/services/dicom_service.py

```
@@ -0,0 +1,219 @@
+        # Bust the study from cache
+        key = f"radiology:dicom:study:{study_uid}"
+        cache.delete(key)
+
+        return {
+            "study_uid": study_uid,
+            "study": fetch_study(dicom_study),
+        }
+
+    except DicomUploadError:
+        raise
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)